Carbon/Functions/Assert-FirewallConfigurable.ps1

```diff
@@ -35,7 +35,7 @@ function Assert-FirewallConfigurable
 
     Use-CallerPreference -Cmdlet $PSCmdlet -Session $ExecutionContext.SessionState
 
-    if( (Get-Service 'Windows Firewall').Status -ne 'Running' ) 

```

Comment:
Keep this function backwards-compatible. Use both service names, please. Check for `Windows Firewall` first, then `MpsSvc`.